### PR TITLE
feat(DMVP-2816): Added new variable for drop namespace logs

### PR DESCRIPTION
**Pull request by aghamyan44**

### CODE REVIEW

File: modules/fluent-bit/values.yaml [hunk 1]
```diff
@@ -15,6 +15,26 @@ config:
...
+        Match kube.*
+        Merge_Log On
+        Keep_Log Off
+        K8S-Logging.Parser On
+        K8S-Logging.Exclude On
+
+    [FILTER]
+        Name          grep
+        Match         *
+        Exclude       $log (ELB-HealthChecker|Amazon-Route53-Health-Check-Service|Health)
```
Comment (aramkarapetian) [inline modules/fluent-bit/values.yaml:31]: Can we also have additional values supplied here to filter out some extra logs?
Also we should here filter prometheus and health with smaller H as seems this is case sensitive.

File: modules/fluent-bit/variables.tf [hunk 2]
```diff
@@ -72,3 +72,9 @@ variable "s3_permission" {
   default     = false
   type        = bool
 }
+
+variable "drop_namespaces" {
+  type        = string
```
Comment (aramkarapetian) [inline modules/fluent-bit/variables.tf:77]: Can we convert this to the list?

File: modules/fluent-bit/values.yaml (hunk 1 above)
Comment (aramkarapetian) [reply modules/fluent-bit/values.yaml:31]: Please have this as a list.

**Review by aramkarapetian — APPROVED** [review]
LGTM

File: variables.tf [hunk 3]
```diff
@@ -113,11 +113,64 @@ variable "log_group_name" {
   default = ""
 }
 
+variable "system_log_group_name" {
```
Comment (mrdntgrn) [inline variables.tf:116]: all this new added variables are for fluent-bit, right?
better to have them in one object variable named for example fluent_bit, somethig like this
variable fluent_bit {
type = object({
  system_log_group_name = string
  values_yaml = string
  ....
})
}

**Review by aramkarapetian — APPROVED** [review]
LGTM

**Pull request merged**